src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationsHearingResultAwardResourceIntTest.kt

```diff
@@ -410,4 +410,378 @@ class AdjudicationsHearingResultAwardResourceIntTest : IntegrationTestBase() {
       }
       """.trimIndent()
   }
+
+  @DisplayName("PUT /adjudications/adjudication-number/{adjudicationNumber}/awards")
```

Comment:
more or less copy of the POST but just tests the create of new awards and no testing of updates and deletes